src/components/GameView/GameCard.vue

```diff
@@ -75,6 +75,10 @@ export default {
       default: '',
       validator: (val) => ['', 'player', 'opponent'].includes(val),
     },
+    highElevation: {
+      type: Boolean,
+      default: false,
+    },
```

Comment:
This is used to set a higher elevation for cards in an overlay. I considered making it a numeric value that get explicitly set to a particular elevation (rather than a high/low boolean), but I think this will be easier for maintaining a consistent style if and when we need cards displayed in other overlays